_i18n/es/resources/user-guides/How-to-mine-Monero-XMR-without-a-mining-equipment.md

```diff
@@ -1,50 +1,29 @@
-{% assign version = '1.1.0' | split: '.' %}
+﻿{% assign version = '1.1.0' | split: '.' %}
 {% include disclaimer.html translated="false" version=page.version %}
```

Comment:
new translation snippet to update: `translated="true"` now

Reply:
So that's a translation snippet only eh? OK no problem, just false to true?

Reply:
true as it is translated